packages/frontend-2/plugins/dayjs.ts

```diff
@@ -2,9 +2,51 @@ import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import localizedFormat from 'dayjs/plugin/localizedFormat'
 import duration from 'dayjs/plugin/duration.js'
+import updateLocale from 'dayjs/plugin/updateLocale'
 
 export default defineNuxtPlugin(() => {
   dayjs.extend(relativeTime)
   dayjs.extend(localizedFormat)
   dayjs.extend(duration)
+  dayjs.extend(updateLocale)
+
+  const customRelativeTime = (date: string): string => {
```

Comment:
DayJS has build-in options to change the formatting of dates, however there are a few issues with it

- There is a default threshold of 28 days for it to become "1 month ago", this threshold is adjustable however it's kind of buggy, quite regularly it would apply the threshold after already rendering. This can be fixed by removing the original threshold first, and applying the new one, however this was really hacky

- For any date bigger than 14 days we just want to output the formatted days, and not relative time, this could be solved with custom formatters, however that required quite a couple extra lines, as well as additional checks for rendering the "ago" text.

Because our use case is quite simple I decided to just create a wrapper function that still relies on DayJS for the calculations, but just handles the custom formattings, as it's a much less hacky/shorter solution